packages/react/src/themes/teams/components/Chat/chatMessageStyles.ts

```diff
@@ -82,13 +89,20 @@ const chatMessageStyles: ComponentSlotStylesInput<
     position: 'absolute',
     right: v.actionMenuPositionRight,
     top: v.actionMenuPositionTop,
-    overflow: p.focused ? 'visible' : 'hidden',
 
-    // hide and squash actions menu to prevent accidental hovers over its invisible area
-    opacity: p.focused ? 1 : 0,
-    width: p.focused ? 'auto' : 0,
-  }),
+    ...(_.isNil(v.showActionMenu) && {
+      overflow: p.focused ? 'visible' : 'hidden',
+      // hide and squash actions menu to prevent accidental hovers over its invisible area
+      opacity: p.focused ? 1 : 0,
```

Comment:
could we augment this comment with a note about `opacity` vs `visibility` strategy - specifically, mention that this affects JAWS behavior on Windows? Just to prevent my future attempts to improve styles by using `visibility` CSS prop instead :)

Reply:
:D sure

Reply:
added